src/nmstate-types/custom-models/NodeNetworkConfigurationInterfaceLLDP.ts

```diff
@@ -0,0 +1,27 @@
+export interface NodeNetworkConfigurationInterfaceLLDP {
+  enabled: boolean;
+  neighbors: {
+    type?: number;
+    subtype?: number;
+    oui?: string;
+    'chassis-id'?: string;
+    'chassis-id-type'?: number;
+    _description?: string;
+    'port-id'?: string;
+    'port-id-type'?: number;
+    'system-name'?: string;
+    'system-description'?: string;
```

Comment:
you can extract this to another type, NeighborInterestingProps, and than you can use keyof instead of creating the NEIGHBOR_INTERESTING_PROPERTIES const with hard-coded values

Reply:
But `NEIGHBOR_INTERESTING_PROPERTIES` is not typescript. It's an array used during runtime

Reply:
I need `NEIGHBOR_INTERESTING_PROPERTIES` to filter the neighbor that has some interesting properties. Some neighbor are just empty of have one property that we don't even want to show